pkg/codegen/dotnet/gen_program.go

```diff
@@ -233,6 +258,44 @@ func (g *generator) genTrivia(w io.Writer, token syntax.Token) {
 	}
 }
 
+func (g *generator) findFunctionSchema(function string) (*schema.Function, bool) {
+	function = LowerCamelCase(function)
+	for _, pkg := range g.program.PackageReferences() {
+		for it := pkg.Functions().Range(); it.Next(); {
+			if strings.HasSuffix(it.Token(), function) {
```

Comment:
Why is this checking via suffix? Can we not use `pkg.Functions().Get(function)`? I'm not a fan of swallowing all errors here. Can we return `(*schema.Function, boo, error)`.

Reply:
Why would it error though? if it errors, that means it couldn't find the function, right? or are there errors that should be propagated back 🤔

Reply:
I believe it will error if it can't acquire the schema, which could happen for many reasons we would want to propagate. If it can confirm that the function isn't there (can acquire the schema, no function there) then we can clearly indicate "The function doesn't exist".